README.md

```diff
@@ -56,20 +56,18 @@ interface Promise
      *
      * If the promise is already resolved, the callback MUST be executed immediately.
      *
-     * @param callable(mixed $reason, mixed $value) @onResolved `$reason` shall be `null` on success, `$value` shall be
-     *     `null` on failure.
+     * @param callable(\Throwable|\Exception|null $exception, mixed $value) @onResolved `$reason` shall be `null` on
+     *     success, `$value` shall be `null` on failure.
      *
-     * @return void
+     * @return mixed Return type and value are unspecified.
```

Comment:
(IMHO `void` is what we want to mean, according to my interpretation of `void` within LSP… but fine. … not going to insist here.)